common/classify.py

```diff
@@ -94,9 +101,19 @@ def test_and_report(self):
         if not os.path.exists(self.results_dir):
             os.makedirs(self.results_dir) 
 
+        logging_interval = 60 # TODO(matthew): Move to config.yaml
+        progress_logger.info("Starting to test on data.")
+        start_time = time.time()
+        last_logged_time = start_time
+
         results = np.empty(shape=(0, TOTAL_SIZE), dtype='S200')
         
         for root, dirs, files in os.walk(self.email_path):
+            curr_time = time.time()
+            if (curr_time - last_logged_time) > logging_interval * 60:
+                progress_logger.info('Exploring directory: {}'.format(root))
```

Comment:
I think 60 minutes is fine for now since if there is activity between the 60 minute intervals, it will also get logged.